circle.yml

```diff
@@ -1,18 +1,71 @@
-dependencies: 
-  cache_directories: 
+# Setup in CircleCI account the following ENV variables:
+# BINTRAY_ACCOUNT
+# BINTRAY_API_KEY
+# DOCKER_USER
+# DOCKER_EMAIL
+# DOCKER_PASSWORD
+general:
+  branches:
+#    only:
+#      - master
+#      - /v[0-9]+(\.[0-9]+)*/
+  artifacts:
+    - /tmp/st2-packages
+
+machine:
+  # Overwrite these ENV variables in parametrized (manual/API) builds
+  environment:
+    ST2_GITURL: https://github.com/StackStorm/st2
+    ST2_GITREV: master
+    ST2_DOCKERFILES_REPO: https://github.com/armab/st2-dockerfiles
+    BUILD_DOCKER: 0
```

Comment:
This temporary disables:
* `st2*` Docker images builds from `st2-dockerfiles`
* deploying created `st2*` Docker containers to Docker Hub
* deploying created packages to Bintray

I'll need all that later.
For now focus on builds.